README.md

```diff
@@ -52,6 +70,14 @@ In your loop function, run the manager loop
 configManager.loop();
 ```
 
+To access your config data within your loop/device code
+
+```cpp
+String name = config.name;
+```
+
```

Comment:
This was completely obvious, I fought myself trying to do this with bad code .. but it was simple -- just putting this up front so others know